test/e2e/k8s/kubernetes.go

```diff
@@ -107,27 +111,62 @@ func (k TestKubernetes) DeleteNamespace(namespace string) {
 	utils.ExpectNoError(err)
 }
 
+func (k TestKubernetes) GetBackup(name, namespace string) *ispnv2.Backup {
+	backup := &ispnv2.Backup{}
+	key := types.NamespacedName{
+		Namespace: namespace,
+		Name:      name,
+	}
+	utils.ExpectNoError(k.Kubernetes.Client.Get(context.TODO(), key, backup))
```

Comment:
The objects in k8s are creating and processed (if they also creating sub-objects and sub-resources) asynchronously.
On fast environments there may not be enough time between steps for the object to be created before checking it.
We should ignore NotFound error there.